apps/back-office/__tests__/api/licenses/route.integration.test.ts

```
@@ -0,0 +1,656 @@
+        where: {
+          id: createdClinicianId,
+        },
+      });
+      await prisma.user.delete({
+        where: {
+          id: mockUserId,
+        },
+      });
+      createdClinicianId = null;
+    }
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow afterEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)